tests/simpleTest/simpleAppState.hpp

```diff
@@ -0,0 +1,139 @@
+// Concord
+//
+// Copyright (c) 2018-2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
```

Comment:
Also can you follow the google coding guidelines and changes the class name to simple_app_state.hpp

Reply:
Fixed